src/shared/views/AppLayout/AppLayout.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { NavigationList } from '../navigation'
+import { TopMenu } from '../topMenu'
+
+import styles from './appLayout.css'
+
+
+const AppLayout = (props) => {
+  return (
+    <div className={styles.main}>
+      <div className={styles.body}>
+        <div className={styles.topMenu}>
+          <TopMenu title={props.location.pathname} />
```

Comment:
I would transform the props.location.pathname here. Or I would pass it down to topmenu as 'path' and then do transformation in `<TopMenu>`